MMM-next-episode.js

```diff
@@ -144,4 +143,5 @@ Module.register("MMM-next-episode", {
     return wrapper;
   }
 });
+
 // @Created By Pierre Gode
```

Comment:
Please remove the author attribution comment as it does not conform to best practices for collaborative projects. Attribution is typically managed through version control systems like Git.